psi4/driver/procrouting/proc.py

```diff
@@ -2691,6 +2695,42 @@ def run_bccd(name, **kwargs):
     optstash.restore()
     return ref_wfn
 
+def run_tdscf_energy(name=None,**kwargs):
```

Comment:
now `e, wfn = energy('pbe', return_wfn=True) \n tddft(wfn)` doesn't work. do people agree that that's what we want for the piecewise access rather than `tddft(ref_wfn=wfn)`?

Maybe this needs two very short functions and one workhorse fn that both call. The inconsistent signatures with `name=None` here and in tddft() make me a little uneasy.

Reply:
in my notions, `run_tdscf_energy(name, **kwargs)` has just that signature (no default for `name`) and has contents https://github.com/psi4/psi4/pull/1885/files#diff-dbf22cd80d16f6d8ef8250145a6d91bfR2698-R2707 and then returns `tdscf(ref_wfn)`.